bin/commands/decryptSecrets.js

```diff
@@ -0,0 +1,79 @@
+/*
+ * Kuzzle, a backend software, self-hostable and ready to use
+ * to power modern apps
+ *
+ * Copyright 2015-2018 Kuzzle
+ * mailto: support AT kuzzle.io
+ * website: http://kuzzle.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable no-console */
+
+const
+  fs = require('fs'),
+  config = require('../../lib/config'),
+  Vault = require('../../lib/api/core/vault'),
+  readlineSync = require('readline-sync'),
+  ColorOutput = require('./colorOutput');
+
+function commandDecryptSecrets (encryptedSecretsFile, options) {
+  const cout = new ColorOutput(options);
+
+  if (!options.vaultKey && !process.env.KUZZLE_VAULT_KEY) {
+    console.log(cout.error('[ℹ] You must provide the vault key with --vault-key <key> or in KUZZLE_VAULT_KEY environment variable'));
+    process.exit(1);
+  }
+
+  let outputFile = options.outputFile;
+  if (!options.outputFile) {
+    outputFile = `${encryptedSecretsFile.split('.enc')[0]}.json`;
```

Comment:
what happens if I store my encrypted configuration in the following file: `/etc/kuzzle.encrypted/config.enc.json`?

You might want to use [path.basename](https://devdocs.io/node~10_lts/path#path_path_basename_path_ext) instead